pipert/core/message_handlers.py

```diff
@@ -71,3 +134,11 @@ def connect(self):
 
     def close(self):
         self.conn.close()
+
+    def _add_offset_to_stream_id(self, stream_id, offset):
+        if stream_id is None:
+            return None
```

Comment:
That might be risky. Might be problematic if the user of the function won't check for the `None`. 

Maybe raising a custom exception might be better? I'm not sure what is the "pythonic" way of in-code exception handling (I have Java's checked exception mechanism in mind). What do you think?

Reply:
In my point of view, the user should already know if he sent `None` to the `stream_id`, if he did, then he also should know that the return value of this function will be `None` because, if you send `None` to a function that manipulate this `None` then it will return `None`.